tests/test_hstrat/test_serialization/test_pop_to_records.py

```diff
@@ -51,7 +51,7 @@ def test_col_to_records(
         )
         for __ in range(popsize)
     ]
-    for birth in range(100):
+    for _ in range(100):
```

Comment:
Use `__` (underscore underscore), which for no specific reason is what is in the rest of the source

Reply:
Fixed in 87cd427.